### DISCUSSION TITLE
array fields defaults not showing when using useFormContext

### DISCUSSION THREAD

**Discussion by dileepthomas** [react-hook-form/react-hook-form#6503] [Q&A]
I am having a use case, where I have multiple tabs, each having a different set of forms, and the save and continue buttons are common for each and every tab. So using useFormContext i am able to enable, disable the button state and on click of buttons, I am getting the formState of each tab forms. But when I use array fields, I have some default array items, when I use useForm it shows but when using useFormContext it doesn't work (Tabs/UserDetails.js)

CSB: [CSB](https://codesandbox.io/s/amazing-jennings-2dvfm)

Any help appreciated

**Comment by bluebill1049** [comment]
```tsx
const { register, reset } = useFormContext({
  // mode: "onBlur",
  //   reValidateMode: "onChange",
  //   defaultValues: {
  //     firstName: ""
  //   }
});
```

https://react-hook-form.com/api/useformcontext
please read the doc, there is no such usage as above. `useFormContext` only brings `useForm` methods as context.
